currency-conversion/src/main/java/lk/ac/ucsc/currencyconversion/CurrencyConversionBean.java

```diff
@@ -0,0 +1,83 @@
+package lk.ac.ucsc.currencyconversion;
+
+public class CurrencyConversionBean {
+    private Long id;
+    private String from;
+    private String to;
+    private double conversionRate;
+    private double quantity;
+
+    public CurrencyConversionBean() {
```

Comment:
@Binurim do u need the empty constructer

Reply:
Nope. Deleted